test/experiments/test_experiment.py

```diff
@@ -69,6 +70,54 @@ async def test_get_experiments_from_scheduler(self, _, __, ___, ____, _____):
         )
         assert test_experiment.experiments == expected_experiments
 
+    @patch(
+        "operationsgateway_api.src.experiments.scheduler_interface.SchedulerInterface.__init__",
+        return_value=None,
+    )
+    @pytest.mark.asyncio
+    async def test_store_experiments(self, _):
```

Comment:
Test could do with a few more comments to explain what is happening.

Reply:
Fixed in 592a500182053d2aff91fdaabb6d09d92ddc6af3